app/helpers/pulfalight_helper.rb

```diff
@@ -23,4 +23,14 @@ def collection_notes_formatter(*_args)
   def component_notes_formatter(*_args)
     html_presenter.collection_notes
   end
+
+  # @return [Class]
+  def aeon_external_request_class
+    Pulfalight::Requests::AeonExternalRequest
```

Comment:
It might be worth enquiring about ensuring that the class used for Aeon requests can be configured somewhere upstream in ArcLight.